### ISSUE TITLE
[Feature idea] open cloze database

### ISSUE THREAD

**Issue by omega3** [zadam/trilium#2172]
Would it be possible add option to be able to open database and close it and open another so that it could function the same way as opening files in other database programs for example Access, LO Base, Excel, etc.

When there is a lot of information stored in database the file becomes large and it can become a problem for slow hardware and the other thing is that it would be easier to keep order when different topics could be in different databases.

For example [cherrytree](https://www.giuspen.com/cherrytree/) works like this.

**Comment by zadam** [comment]
Hi, it's a different philosophy compared to e.g. cherrytree. Different topics etc. are still expected to be in the same database file.

Having said that, you can switch databases by pointing to a different directory in env. variable - see https://github.com/zadam/trilium/wiki/Data-directory

**Comment by omega3** [comment]
I believe that adding such option would not hurt this philosophy at all. Some users would prefer to have all in one database, some would prefer to separate topics into different databases. Different databases would allow to share databases. For example someone, maybe a teacher or student, may want to have private and public databases with different content. Such program allows to share knowledge the same way as books, but it is easier to make such database than an ebook.
 
I think default database could play the role of main database that match your philosophy. Ability to easily access different databases would satisfy other philosophical approaches.

**Comment by zadam** [comment]
Yes, but supporting different philosophies can take a lot of time and effort. In my view implementing what you suggest does not allow any new use case, it's _merely_ a different way to do what is already possible. As such it always seems to me better to spend my (rare) time on features which bring something new to the project.
